src/sentry/notifications/activity/release.py

```diff
@@ -157,29 +158,23 @@ def get_participants(self):
                 or NotificationSettingOptionValues.COMMITTED_ONLY.value  # product default
             )
 
-        # filter down to members which have been seen in the commit log:
-        participants_committed = {
-            user: GroupSubscriptionReason.committed
-            for user, option in users_with_options.items()
-            if (
+        users_to_reasons: MutableMapping[User, int] = {}
```

Comment:
does this logic change belong in this PR? seems unrelated to moving files around

Reply:
The logic isn't changing but I had to the refactor because the type checker doesn't like  `itertools.chain`.